packages/web/src/components/home/card-list/CardList.tsx

```diff
@@ -2,44 +2,69 @@ import IconTriangleArrowUp from "@components/common/icons/IconTriangleArrowUp";
 import IconTriangleArrowDown from "@components/common/icons/IconTriangleArrowDown";
 import { CardListWrapper, ListItem } from "./CardList.styles";
 import DoubleLogo from "@components/common/double-logo/DoubleLogo";
-import { CardListPoolInfo, CardListTokenInfo } from "@models/common/card-list-item-info";
+import {
+  CardListKeyStats,
+  CardListPoolInfo,
+  CardListTokenInfo,
+} from "@models/common/card-list-item-info";
 import { useCallback, useMemo } from "react";
 import { SwapFeeTierInfoMap } from "@constants/option.constant";
 import MissingLogo from "@components/common/missing-logo/MissingLogo";
 
 interface CardListProps {
-  list: Array<CardListTokenInfo | CardListPoolInfo>;
+  list: Array<CardListTokenInfo | CardListPoolInfo | CardListKeyStats>;
   onClickItem: (id: string) => void;
   isHiddenIndex?: boolean;
 }
 
-function isTokenInfo(info: CardListTokenInfo | CardListPoolInfo): info is CardListTokenInfo {
+const enum CardType {
+  "keyStats",
+  "token",
+  "pool",
+}
+
+function typeTokenInfo(
```

Comment:
The function name looks like a variable name. What if we change it to a verb?

Reply:
DONE